s3/pkg/datastore/azure/azure.go

```diff
@@ -331,6 +332,76 @@ func (ad *AzureAdapter) ListParts(ctx context.Context, multipartUpload *pb.ListP
 	return nil, ErrNotImplemented
 }
 
+func (ad *AzureAdapter) BackendCheck(ctx context.Context, backendDetail *pb.BackendDetailS3) error {
```

Comment:
@vineela1999 can we replace this code by re-using code ?
question is

- can backendCheck be an aggregate API ?
- it can call existing methods putObject, deleteObject to do the check

Reply:
Yes done thanks